cli/tests/std_tests.rs

```diff
@@ -1,34 +1,32 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-extern crate lazy_static;
-extern crate tempfile;
-use deno_cli::test_util::*;
-use std::env;
-use std::process::Command;
-use tempfile::TempDir;
 
-#[test]
-fn std_tests() {
-  // TODO: this is hacky and should conditionally skipped
-  // some more idiomatic way
-  // Run only with release build
-  if let Ok(mode) = env::var("DENO_BUILD_MODE") {
-    if mode == "release" {
-      let dir = TempDir::new().expect("tempdir fail");
-      let mut deno_cmd = Command::new(deno_exe_path());
-      deno_cmd.env("DENO_DIR", dir.path());
+// Runs only on release build
```

Comment:
Please add `// TODO fix tests in release mode.`

Reply:
In debug mode? Working in release

Reply:
er, yes that's what I meant.